docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Router Deployment
+---
+
+import CliLogin from '/docs/_cli-login.md'
+import LinuxPackageRepo from '/docs/_linux-package-repo.mdx';
+import CliLogFormats from '/docs/_cli-log-levels-and-formats.md'
+
+This article is about deploying a Ziti router as a Linux system service. [This page](/learn/introduction/03-components.md#openziti-router) provides an overview of the architecture.
+
+Routers provide Ziti network entry and exit points for clients and smart routing through the mesh. You must have at least one router that is advertising an edge connection to clients. If you have more than one router, you must configure transport links between them to form the fabric mesh.
```

Comment:
feels too reference-y to me. I like the overall vibe but I'd like to steer away from things like "you must configure transport links between them to form the fabric mesh." since this is actually entirely untrue. it's not necessary at all... :) so when it's squishy like that, i vastly prefer not to even open that door. I think this whole section could mabye even be just removed? i feel like these nuances are a whole page (or set of pages) on their own

Reply:
I think I see what you mean. With multiple routers, it is possible for them to provide only the edge without any fabric. Services will still work as long as there's one common, online router between the dial and bind identities. Smart routing via the mesh fabric is, technically, optional.